### ISSUE TITLE
API v2: Strings are not properly rendered

### ISSUE THREAD

**Issue by jmcanterafonseca** [telefonicaid/fiware-orion#1005]
The following test fails:

```
--NAME--
GET Entities V2 string

--SHELL-INIT--
dbInit CB
brokerStart CB

--SHELL--

#
# 01. POST /v1/updateContext
# 03. GET /v2/entities
#

echo "01. POST /v1/updateContext"
echo "=========================="
payload='{
  "contextElements": [
    {
      "id":   "8787ZZZ",
      "type": "Car",
      "attributes" : [
        {
          "name" : "speed",
          "type": "number",
          "value" : 150
        },
        {
          "name" : "brand",
          "type": "string",
          "value" : "Mercedes Benz"
        },
        {
          "name" : "model",
          "type": "string",
          "value" : "Klasse C"
        }
      ]
    }
  ],
  "updateAction": "APPEND"
}'
orionCurl --url /v1/updateContext --payload "$payload" --json
echo
echo

echo "02. GET /v2/entities"
echo "===================="
orionCurl --url /v2/entities --json
echo
echo

--REGEXPECT--

01. POST /v1/updateContext
==========================
HTTP/1.1 200 OK
Content-Length: 1854
Content-Type: application/json
Date: REGEX(.*)

"contextElements": [
    {
      "id":   "8787ZZZ",
      "type": "Car",
      "attributes" : [
        {
          "name" : "speed",
          "type": "number",
          "value" : 150
        },
        {
          "name" : "brand",
          "type": "string",
          "value" : "Mercedes Benz"
        },
        {
          "name" : "model",
          "type": "string",
          "value" : "Klasse C"
        },
      ]
    }
  ]


02. GET /v2/entities
====================
HTTP/1.1 200 OK
Content-Length: 232
Content-Type: application/json
Date: REGEX(.*)

[
    {
      "id": "8787ZZZ",
      "type": "Car",
      "speed": 150,
      "brand": "Mercedes Benz",
      "model": "Klasse C"
    }
]


--TEARDOWN--
brokerStop CB
dbDrop CB

```

**Comment by fgalan** [comment]
Current version renders:

```
        {
          "name" : "speed",
          "type": "number",
          "value" : 150
```

as 

```
        {
          "name" : "speed",
          "type": "number",
          "value" : "150"
```

In order to implement this, we first has to implement this (at mongoBackend layer): https://github.com/telefonicaid/fiware-orion/issues/863.

**Comment by fgalan** [comment]
Fixed in PR #1008
